maps/_common/areas/station/research.dm

```diff
@@ -35,6 +35,10 @@
 	icon_state = "research"
 	lightswitch = TRUE
 
+/area/rnd/secure_hallway
+	name = "Research - Secure Hallway"
+	icon_state = "research"
+	lightswitch = TRUE
```

Comment:
```suggestion
	lightswitch = TRUE

```

Reply:
What is the change here? I'm not seeing one.

Reply:
it adds a new line for spacing.